routing/route.cpp

```diff
@@ -185,6 +188,22 @@ bool Route::GetNextTurn(double & distanceToTurnMeters, turns::TurnItem & turn) c
   return true;
 }
 
+bool Route::GetNextTurns(vector<turns::TurnItemDist> & turns) const
+{
+  turns.clear();
+
+  turns::TurnItemDist currentTurn;
+  if (!GetCurrentTurn(currentTurn.m_distMeters, currentTurn.m_turnItem))
+    return false;
+  turns.emplace_back(move(currentTurn));
+
+  turns::TurnItemDist nextTurn;
+  if (!GetNextTurn(nextTurn.m_distMeters, nextTurn.m_turnItem))
+    return true;
```

Comment:
Yes. The thing is here we have one correct turn in turns. It could happen at the end of the route.